src/overlays/actors/ovl_En_Gamelupy/z_en_gamelupy.h

```diff
@@ -8,10 +8,14 @@ struct EnGamelupy;
 typedef void (*EnGamelupyActionFunc)(struct EnGamelupy*, PlayState*);
 
 typedef struct EnGamelupy {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x4C];
-    /* 0x0190 */ EnGamelupyActionFunc actionFunc;
-    /* 0x0194 */ char unk_194[0x10];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ ColliderCylinder collider;
+    /* 0x190 */ EnGamelupyActionFunc actionFunc;
+    /* 0x194 */ s32 rupeeIndex;
+    /* 0x198 */ s16* unk_198;
```

Comment:
My guess is that this is actually an Actor*, but we should probably wait for more info from `EnLookNuts`. Can you add a `//! TODO: Look at type based on EnLookNuts decomp`?